src/screens/IndexScreen.js

```diff
@@ -416,6 +419,14 @@ export const IndexScreen = ({ navigation, route }) => {
                           hasSectionHeader={false}
                         />
                       )}
+                      {query === QUERY_TYPES.CATEGORIES &&
+                        (!!categoryListIntroText || !!texts.categoryList.intro) && (
+                          <Wrapper>
+                            <RegularText>
+                              {categoryListIntroText || texts.categoryList.intro}
+                            </RegularText>
+                          </Wrapper>
+                        )}
```

Comment:
this should be ListHeaderComponent of the `CategoryList` or how does that work?
you have addd the usage of `ListHeaderComponent={ListHeaderComponent}` in `CategoryList` but are not passing it here in line 419

Reply:
In line 419, the `query` is named `pointOfInterest`. The header here is only used if there is a subcategory. Therefore, there is no situation that will affect the `intro` text.

I am sharing screenshots.

|categoryList|subCategoryList|
|--|--|
![Simulator Screen Shot - iPhone 14 Plus - 2023-02-10 at 16 24 32](https://user-images.githubusercontent.com/11755668/218130037-dba3c47b-8c3a-4387-bd2d-676c48447bc2.png) | ![Simulator Screen Shot - iPhone 14 Plus - 2023-02-10 at 16 24 42](https://user-images.githubusercontent.com/11755668/218130005-03353913-81e7-4383-b6e7-8f30c52461b4.png)

Reply:
ok 🤞